### ISSUE TITLE
Footer text color has bad contrast ratio

### ISSUE THREAD

**Issue by mattgaspar** [emailmonday/Cerberus#243]
**Describe the bug**
Footer text color has bad contrast ratio, fails accessibility check in Chrome DevTools (see screenshot) 

**To Reproduce**
1. Which template? All 3
2. Which email clients and/or devices are affected? 
Using Outlook on iOS in Dark mode the text actually disappears because of the low contrast ratio (maybe also a bug in the Outlook app).  (Haven't tested with other email clients in dark mode)

**Screenshots**
![image](https://user-images.githubusercontent.com/811116/84438071-cb50f980-abea-11ea-8381-d475d4abac27.png)

**Comment by TedGoas** [comment]
Good call, best to set a good example! Updating shortly.
